modules/library/main/src/main/java/org/geotools/styling/visitor/DpiRescaleStyleVisitor.java

```diff
@@ -62,15 +62,6 @@ protected Expression rescale(Expression expr) {
         return RescalingMode.Pixels.rescaleToExpression(scale, v);
     }
     
-    @Override
```

Comment:
Shouldn't we have a method working on the list of expressions, with the same logic here?
If the build is not failing, I guess I'm either missing something, or we lack a test covering this functionality.